main.go

```diff
@@ -100,6 +102,8 @@ func main() {
 	if err := server.Shutdown(ctx); err != nil {
 		log.Fatal(err)
 	}
+
+	util.ClosePackageWatcher()
```

Comment:
What about using a `defer util.ClosePackageWatcher()` on line 77 instead? Like this we can keep the logic together and the watcher is not closed when it is not started.

Reply:
Fixed.